tests/unit/lib/tasks/BackbeatTask.spec.js

```diff
@@ -0,0 +1,42 @@
+const assert = require('assert');
+
+const werelogs = require('werelogs');
+
+const BackbeatTask = require('../../../../lib/tasks/BackbeatTask');
+
+const logger = new werelogs.Logger('BackbeatTask:test');
+
+describe('BackbeatTask', () => {
+    it('BackbeatTask.retry() should handle and trace double callback (S3C-4457)', done => {
```

Comment:
why do we need a ref to the ticket here?

Reply:
Because the fix is actually a workaround, I anticipate that we would remove the workaround (and this test) once we find and fix the root cause, hence leaving the ticket number here can help to track this.

Reply:
Would it make sense to have that information as a comment (TODO)? It looks a little odd in the test description!

Reply:
Indeed it could be better, will do